src/solvers/bitwuzla/bitwuzla_conv.cpp

```diff
@@ -25,11 +25,11 @@ smt_convt *create_new_bitwuzla_solver(
 bitwuzla_convt::bitwuzla_convt(const namespacet &ns, const optionst &options)
   : smt_convt(ns, options), array_iface(true, true), fp_convt(this)
 {
-  bitw = bitwuzla_new();
-  bitwuzla_set_option(bitw, BITWUZLA_OPT_PRODUCE_MODELS, 1);
+  bitw_options = bitwuzla_options_new();
```

Comment:
The destructor should probably also get rid of this new object.

Reply:
Implemented as suggested.